### ISSUE TITLE
can't use 2022.09.19.7 on php 7.4.3 / fpm

### ISSUE THREAD

**Issue by ignoreigor** [tine20/tine20#7396]
Getting this error when using setup.php regardless wheter via console nor browser.
Subdir-Install / php 7.4.3 fpm in nginx

Maybe a consequence of  "feature(PHP8.1 Support)" or "fix(zend1/php8.1) temporarily fix github reference"

```
root@# sudo -u www-data php setup.php  --update
PHP Fatal error:  Uncaught Error: Class 'Zend\Http\PhpEnvironment\Request' not found in /var/www/tine20/Setup/Core.php:97
Stack trace:
#0 /var/www/tine20/setup.php(14): Setup_Core::dispatchRequest()
#1 {main}
  thrown in /var/www/tine20/Setup/Core.php on line 97
```

**Comment by ignoreigor** [comment]
Still occurs in 2022.09.20.40
Maybe #7399 is similar.

**Comment by pschuele** [comment]
yup, we are aware of this bug. it is in our build scripts - when you run "composer install" in your tine dir, it should work.
we'll update / close this issue as soon as a fixed release is available and will remove all broken releases.

**Comment by ignoreigor** [comment]
> 



> when you run "composer install" in your tine dir, it should work.

Yupp, it works as you said, but I have to create composer.json in tine's directory. Meanwhile it crashes my tzdata which throws "**PHP Fatal error:  DateTime::__construct(): Timezone database is corrupt - this should *never* happen!** - but that was only a sideeffect and not caused by tine.

**Comment by pschuele** [comment]
the problem is fixed in current release 2022.09.22.8
